bin/iron/src/app.rs

```diff
@@ -148,6 +225,9 @@ fn on_menu_event(event: WindowMenuEvent) {
         "close" => {
             event.window().close().unwrap();
         }
+        "settings" => {
+            event.window().emit("settings", ()).unwrap();
+        }
```

Comment:
this is a very unclear name as well. `settings` is not an action. if I understood from context, `show-settings` would be better?

Reply:
My reasoning is that the function is the handler for menu events `on_menu_event`, so "settings" is clicking the "Settings" button. It made sense, but I'm happy to change it.

Reply:
yes, and from that perspective it looks logic. but these are still global events, not a specific channel about menu events.


`
we could also do something different and namespace these to make it clear where they came from: `emit("menu:settings")`

Reply:
Yap 👍 that makes sense